S-L-Murphey/Gamer-Rater-API
# GamerRater Ch. 9

#### Changes Made


1. Added file `category.py` to `views` and updated the URLs 


#### Steps to Review

1. Checkout to this branch locally
   ```
   git fetch --all
   git checkout slm-gamerrater-9
   ```
2. Ensure the backend and client are both running.
4. Test app functionality.

   > Example: When viewing the client, the user should be able to see all games and games link, clicking on the games should reveal all the details on a game and clicking register new game should allow one to create a new game.
   
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
